contrib/seeds/makeseeds.py

```diff
@@ -19,13 +19,10 @@
 
 # These are hosts that have been observed to be behaving strangely (e.g.
 # aggressively connecting to every node).
-SUSPICIOUS_HOSTS = {
-    "130.211.129.106", "178.63.107.226",
-    "83.81.130.26", "88.198.17.7", "148.251.238.178", "176.9.46.6",
-    "54.173.72.127", "54.174.10.182", "54.183.64.54", "54.194.231.211",
-    "54.66.214.167", "54.66.220.137", "54.67.33.14", "54.77.251.214",
-    "54.94.195.96", "54.94.200.247"
-}
+f = open("suspicious_hosts.txt")
+SUSPICIOUS_HOSTS = {x if "\n" not in x else x.replace("\n", "") for x in f.readlines()}
+f.close()
```

Comment:
Nit: Slightly shorter using `strip()` and RAII:

```
with open("suspicious_hosts.txt") as f:
    SUSPICIOUS_HOSTS = {s.strip() for s in f if s.strip()}
```

Reply:
> Nit: Slightly shorter using `strip()` and RAII:
> 
> ```
> with open("suspicious_hosts.txt") as f:
>     SUSPICIOUS_HOSTS = {s.strip() for s in f if s.strip()}
> ```

Didn't know that strip() would work in this case, just changed it